storage/bootstrap/bootstrapper/commitlog/source.go

```diff
@@ -49,8 +50,20 @@ type encoder struct {
 }
 
 type encoderMap struct {
```

Comment:
nit: rename this to something series specific, `seriesEncodersByTime` or some such